app/views/RoomsListView/index.tsx

```diff
@@ -561,6 +575,8 @@ class RoomsListView extends React.Component<IRoomsListViewProps, IRoomsListViewS
 				// @ts-ignore
 				this.state.chatsUpdate = chatsUpdate;
 				// @ts-ignore
+				this.state.omnichannelsUpdate = omnichannelsUpdate;
```

Comment:
Why not use this.setState({})?

Reply:
Because this is a direct state mutation when the screen wasn't mounted yet. And I keep the pattern from the screen. What do you think about this? @GleidsonDaniel